README.md

```diff
@@ -35,6 +35,7 @@ Creating this web application will provide exposure to:
 Model -
```

Comment:
Also, give a points system to your quality rubric unless you consider all items weighted equally.